simple_history/tests/models.py

```diff
@@ -25,6 +25,14 @@ def get_absolute_url(self):
         return reverse("poll-detail", kwargs={"pk": self.pk})
 
 
+class PollWithNonEditableField(models.Model):
+    question = models.CharField(max_length=200)
+    pub_date = models.DateTimeField("date published")
+    created = models.DateTimeField(auto_now_add=True, editable=False)
```

Comment:
By making this field `auto_now_add`, it is only touched at create time.  If you change it to `auto_now` it will better test the behavior.